docs/src/administration/users.md

```diff
@@ -10,40 +10,44 @@ description: |
 
 ## User roles
 
-At the project level:
+{{< note theme="warning" title="Important" >}}
+We are deprecating granting user roles on individual environments. You should use environment types instead.
```

Comment:
@GuGuss what's the timeline to deprecate user roles on individual environments?

Reply:
The next automated update of our Git component will deprecate support for setting user permissions on individual environments. It is happening in the coming weeks as the blog post is saying: https://platform.sh/blog/2021/environment-types-simpler-user-permissions/